<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.linecorp.bot.client.ExceptionConverterTest" tests="3" skipped="0" failures="0" errors="0" timestamp="2017-10-30T13:46:33" hostname="Lenovo-PC" time="0.01">
  <properties/>
  <testcase name="convertTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.001"/>
  <testcase name="convertUnknownExceptionTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.0"/>
  <testcase name="exceptionInConvertFallbackedTest" classname="com.linecorp.bot.client.ExceptionConverterTest" time="0.008"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
